Governance.md

```diff
@@ -0,0 +1,84 @@
+# Governance of rules_scala
+
+[bazelbuild/rules_scala](https://github.com/bazelbuild/rules_scala) is under the [bazelbuild](https://github.com/bazelbuild) org
+but is governed by contributors from the community.
+
+## Goals of this document
+
+To clarify:
+
+1. Values of the project.
+2. How to get a PR merged
+3. How long you should need to wait for a review
+4. How to resolve conflicts in project direction
+5. How to make larger and potentially disruptive changes
+
+## Values of rules_scala
+
+1. **Correctness**: Bazel values fully reproducible builds which are reliable and hermetic (only depend on the current state of your repository)
```

Comment:
"correct" can be interpreted in different ways, so "Repeatability" may be a better term?

Reply:
This PR is irrelevant. Please do not comment on the text. There is an attached google doc. Comment on that.